index.js

```diff
@@ -26,50 +26,91 @@ class MainSection extends Component {
 }
 
 class Currency extends Component {
```

Comment:
@masklinn thank you so much for the contribution! 👍 

I have a doubt though, on Github I'm getting this appearance for the diff:
<img width="1010" alt="screen shot 2018-07-19 at 20 31 18" src="https://user-images.githubusercontent.com/11976836/42962841-d4c9c14c-8b92-11e8-9ccc-4565aafaa5ac.png">

Any idea why it's showing the changed lines in dark red?
Also, can you kindly provide a screenshot of how the end result looks?